libtextsecure/test/contacts_parser_test.js

```diff
@@ -93,7 +93,7 @@ describe("GroupBuffer", function() {
       assert.strictEqual(group.name, "Hackers");
       assertEqualArrayBuffers(group.id.toArrayBuffer(), new Uint8Array([1,3,3,7]).buffer);
       assert.sameMembers(group.members, ['cereal', 'burn', 'phreak', 'joey']);
-      assert.strictEqual(group.avatar.contentType, "image/jpg");
+      assert.strictEqual(group.avatar.contentType, "image/jpeg");
```

Comment:
Based on my research `image/jpg` is not a valid MIME type, so I updated our tests to make our code easier.

Reply:
Have you tested it against your phone? Also, you should get an Android device to test against if you don't already have one.

Reply:
Yes, I tested against my iOS main device. I plan to get an Android device for testing as well.

Reply:
Do you have any tips on how to better test this particular change?